### ISSUE TITLE
Too much helium crashes iOS Safari 

### ISSUE THREAD

**Issue by fregante** [arturbien/balloons#1]
If I release too many balloons, my Safari tab crashes. How can I release more balloons?

**Comment by arturbien** [comment]
@fregante yeah this lib was not meant for releasing hundreds of balloons. Tried to make it so that it does just enough. I might add the option to customise the number of balloons if it gets more usage though ✨
